### ISSUE TITLE
Editor dropdowns close when right clicking on them in Firefox

### ISSUE THREAD

**Issue by DaazKu** [vanilla/vanilla#5208]
https://github.com/vanilla/vanilla/pull/5061/files#diff-4cdf1a850be5a6b9c69af45f60dbf08bR356

Firefox send a click event on right clicks. 
Chrome does not.
We should check that IE and Safari behave correctly.

EDIT: The dropdown should not close when we right click on it.

**Comment by slafleche** [comment]
Do we want it to close? What's the use case for left clicking in Vanilla? I don't really see why this is a bug. Browsers behave differently and users of that browser are used to their quirks.

**Comment by slafleche** [comment]
In PR: https://github.com/vanilla/vanilla/pull/5619

**Comment by Charrondev** [comment]
I've looked into this a little bit but it doesn't seem like it's worth doing this as part of the current. It seems to only be something that would improve things for developers slightly and brings a lot of risk. Our editor flyouts are pretty messy and originate from more places than is easy to keep track of.

Because the events are refired with `$.trigger()` none of the actual context of the event gets passed along with it to the final handler either so this fix would need to be put in numerous places right now.

In summary this probably shouldn't be handled until after we change or refactor our flyouts.

**Comment by charrondev** [comment]
Non-security fixes will no longer be prioritized for advanced editor by the vanilla team. Rich Editor ships with Vanilla 2.8 as the default. https://github.com/vanilla/vanilla/pull/8302
